bench/blas_benchmark.hpp

```diff
@@ -47,8 +51,37 @@ ScalarT *new_data(size_t size, bool initialized = true) {
   }
   return v;
 }
+
+template <typename ScalarT>
+ScalarT *new_const_data(size_t size, ScalarT value = 0) {
+  ScalarT *v = new ScalarT[size];
```

Comment:
do we need a raw pointer? can we use unique_ptr at least?

Reply:
It turns out that we don't actually need these functions at all - I think they're left over from an earlier refactoring pass. Removing.